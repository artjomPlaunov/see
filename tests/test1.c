int main() {}

int func  (

    )


{      }


int   


         ImaFunction () {
         }
