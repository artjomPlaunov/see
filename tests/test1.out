(Ast.Prog
   [(Ast.Fun (Ast.INT_T, (Ast.ParamLst []), (Ast.Ident "main"),
       (Ast.Block [])));
     (Ast.Fun (Ast.INT_T, (Ast.ParamLst []), (Ast.Ident "func"),
        (Ast.Block [])));
     (Ast.Fun (Ast.INT_T, (Ast.ParamLst []), (Ast.Ident "ImaFunction"),
        (Ast.Block [])))
     ])